build/media_source/com_templates/vue/components/app.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+	<div>
+		<v-navigation-drawer v-model="showSettings" app disable-resize-watcher id="Settings" class="settings">
+			<h2>{{ translate('COM_TEMPLATES_SETTINGS') }}</h2>
+			<hr>
+			<component :is="selectedSettings" class="form-group" :grid='grid_selected' :column='column_selected'
+					   @reset="reset"></component>
+		</v-navigation-drawer>
+
+		<v-content class="pagebuilder">
+			<h2>{{ translate('COM_TEMPLATES_VIEW') }}</h2>
+			<form name="global-settings">
+				<div class="form-group">
+					<label for="grid-size">{{ translate('COM_TEMPLATES_GRID_SIZE') }}</label>
+					<input v-model.number="gridSize" type="number" id="grid-size" name="grid-size" class="form-control">
+				</div>
+			</form>
+
+			<div v-for="grid in gridArray" :key="grid.id" class="row-wrapper">
+				<grid-element :grid="grid" :grid-size="gridSize"
+							  @editColumn="editColumn"
+							  @addElement="addElement"
+				>
+				</grid-element>
+
+				<div class="btn-wrapper">
+					<button type="button" class="btn btn-lg" @click="editGrid(grid)">
+						<span class="icon-options"></span>
+						<span class="sr-only">{{ translate('COM_TEMPLATES_EDIT_GRID') }}</span>
+					</button>
+					<button type="button" class="btn btn-lg" @click="deleteGrid(grid)">
+						<span class="icon-cancel"></span>
```

Comment:
`icon-cancel`   change it to delete icon. Because, you are not canceling any think you are deleting.

Reply:
Make try to make icons small

Reply:
This is a class from fontawesome which Joomla is using to display that icon.
It was an issue that the icons were too small. People could have problems with small icons.